### PR TITLE
Bay/Polaris loadouts

### PR DESCRIPTION
**Pull request by tigercat2000**
:cl:
add: There's a new tab in the character setup screen, "loadouts". This allows you to spawn with a limited number of preset items.
tweak: Character setup now uses #defines for the different tabs.
/:cl:

TODO:
 - [x] Fix merge conflicts.
 - [x] Add skirts to the map.
 - [x] Figure out a way to make job skirts better, only cost like 3 points and you choose which departments you want or something.
 - [x] Figure out the initial loading points we are going to go with. 10 points is a bit high. Made 5.
 - [x] Add "Job Restriction" column to the UI, so it's obvious what job an item is limited to.

Basics
 - Allows you to select up to 10 items in the character menu
 - Saved per-character
 - The items will be spawned when you join
 - Some items may have job limitations

Included Items
 - Accessories
   - Scarf
   - Black Scarf
   - Christmas Scarf
   - Dark Blue Scarf
   - Green Scarf
   - Light Blue Scarf
   - Orange Scarf
   - Purple Scarf
   - Red Scarf
   - Striped Blue Scarf
   - Striped Green Scarf
   - Striped Red Scarf
   - White Scarf
   - Yellow Scarf
   - Zebra Scarf
 - Cosmetics
   - Blue lipstick
   - Jade lipstick
   - Purple lipstick
   - Red lipstick
 - General
   - d20
 - Uniforms and Casual Dress
   - Blue Plaid Skirt
   - Purple Plaid Skirt
   - Red Plaid Skirt
   - Atmospherics Skirt
   - Black Skirt
   - Cargo Skirt
   - Chemist Skirt
   - CMO Skirt
   - Engineer Skirt
   - HOS Skirt
   - Medical Skirt
   - QM Skirt
   - Roboticist Skirt
   - Scientist Skirt
   - Security Skirt
   - Virologist Skirt
   - Warden Skirt

# FEEL FREE TO SUGGEST OTHER ITEMS TO RIP FROM POLARIS
I left out a fair amount because I ran out of time to filter them.
https://github.com/PolarisSS13/Polaris/tree/master/code/modules/client/preference_setup/loadout

### CODE REVIEW

**Comment by Spacemanspark** [comment]
The blue hoodie from Polaris would be neat.

**Comment by LittleBigKid2000** [comment]
Needs to include zippo lighters and flasks
Idea totally not stolen from TechDragon

**Comment by NTSAM** [comment]
all dem hoodie

**Comment by Aurorablade** [comment]
the ur-scarf..that shows up in examine as every scarf at once.

...Oh and some bandanas..my tarajan likes bandanas...

**Comment by KasparoVy** [comment]
Eyepatch?

**Comment by Iamgoofball** [comment]
instead of adding 3000+ lines why dont you just add a costume vendor in the dorms that vends all this shit

**Comment by Fox-McCloud** [comment]
>instead of adding 3000+ lines why dont you just add a costume vendor in the dorms that vends all this shit

We do--we have a total of 5. Generic clothing, public costume vendor, jumpsuits, shoes, and hats--apparently that's still not enough.

File: code/modules/client/preference/loadout/loadout_uniform.dm [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	display_name = "plaid skirt, red"
+	path = /obj/item/clothing/under/dress/plaid_red
+
+/datum/gear/uniform/skirt/black
+	display_name = "skirt, black"
+	path = /obj/item/clothing/under/blackskirt
+
+/datum/gear/uniform/skirt/ce
+	display_name = "skirt, ce"
+	path = /obj/item/clothing/under/rank/chief_engineer/skirt
```
Comment (Fox-McCloud) [inline code/modules/client/preference/loadout/loadout_uniform.dm]: Let me wear a dress as a CE; it totally won't get caught in the gears, equipment or anything like that; totes not a safety hazard.

File: code/modules/clothing/under/jobs/security.dm [hunk 2]
```diff
@@ -29,6 +37,14 @@
 	flags = ONESIZEFITSALL
 	strip_delay = 50
 
+/obj/item/clothing/under/rank/security/skirt
+	name = "security officer's jumpskirt"
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/jobs/security.dm]: This is so lulzy on so many levels as to be laughable ridiculous.

**Comment by HereticChurch** [comment]
This is a nice system :+1: :x: :100: if just for more skirts.

File: code/modules/client/preference/loadout/loadout_uniform.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/client/preference/loadout/loadout_uniform.dm]: cosmetics, shouldn't matter that much

File: code/modules/clothing/under/jobs/security.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/modules/clothing/under/jobs/security.dm]: cosmetics, shouldn't matter

**Comment by FalseIncarnate** [comment]
I like it, but i'd make the jumpsuits(skirts) cost more than the default 1 point. The default number of points also seems high, maybe drop that to 5.

If you drop the starting amount to 5, maybe make the jumpsuits cost 2-3 points

**Comment by theColdflame** [comment]
:+1: to the system in general, but shouldn't alt job clothing probably just go in the relevant locker?

**Comment by Gummin** [comment]
I think the system is interesting and even a cool idea, I just feel like that it needs to justify that it isn't just "unneeded bulk", and I don't know if it will be able to do that.

File: code/modules/client/preference/loadout/loadout.dm [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+/hook/startup/proc/populate_gear_list()
+	//create a list of gear datums to sort
+	for(var/geartype in subtypesof(/datum/gear))
+		var/datum/gear/G = geartype
+
+		var/use_name = initial(G.display_name)
+		var/use_category = initial(G.sort_category)
+
+		if(G == initial(G.subtype_path))
```
Comment (PsiOmegaDelta) [inline code/modules/client/preference/loadout/loadout.dm]: Now that's a useful trick for filtering categorization entries out. Going to remember that one.
Just wondering, will an -instance- of ```/datum/gear``` really match the -type- ```/datum/gear``` or should it actually be ```G.type == initial(G.subtype_path)```?

File: code/game/jobs/job_controller.dm [hunk 4]
```diff
@@ -464,6 +515,22 @@ var/global/datum/controller/occupations/job_master
 							H.equip_to_slot_or_del(BPK, slot_back,1)
 					H.species.equip(H)
 
+			//Deferred item spawning.
+			if(spawn_in_storage && spawn_in_storage.len)
+				var/obj/item/weapon/storage/B
+				for(var/obj/item/weapon/storage/S in H.contents)
+					B = S
+					break
```
Comment (PsiOmegaDelta) [inline code/game/jobs/job_controller.dm]: I assume you were looking at Polaris' implementation?
It's a bit dated compared to Bay's that, among other things, first checks that the item actually fits in the found storage (and if that fails tries to put the item in hand, etc.).

**Comment by tigercat2000** [comment]
> instead of adding 3000+ lines why dont you just add a costume vendor in the dorms that vends all this shit

i added maybe 100-250 lines of actual code
the massive line changes is from reorganizing files

File: code/modules/client/preference/loadout/loadout.dm (hunk 3 above)
Comment (tigercat2000) [inline code/modules/client/preference/loadout/loadout.dm]: subtypesof returns a list of paths, the actual typing on `G` is just for initial() to work; subtype_path is just a var containing a path

Comment (PsiOmegaDelta) [inline code/modules/client/preference/loadout/loadout.dm]: This is true, I'm simply not awake.

**Comment by tigercat2000** [comment]
@FalseIncarnate  
> I like it, but i'd make the jumpsuits(skirts) cost more than the default 1 point. The default number of points also seems high, maybe drop that to 5.

> If you drop the starting amount to 5, maybe make the jumpsuits cost 2-3 points

the skirts are cheap because most of them are limited to a certain job
if you want skirts on all of the jobs you'll have to spend 7 points

File: code/game/jobs/job_controller.dm (hunk 4 above)
Comment (tigercat2000) [inline code/game/jobs/job_controller.dm]: alright i'll check out bay's

**Comment by tigercat2000** [comment]
@theColdflame 
> :+1: to the system in general, but shouldn't alt job clothing probably just go in the relevant locker?

probably, but this has the mild advantage of "guaranteed to get it" unless you like actually include enough for all of that department's job slots

**Comment by tigercat2000** [comment]
@Fox-McCloud 
> We do--we have a total of 5. Generic clothing, public costume vendor, jumpsuits, shoes, and hats--apparently that's still not enough.

that's not the point
those all have intentional stock limitations on all of their items

the system in general is useful for more than that; the demo items are just good examples of mild customization shit to do

**Comment by Fox-McCloud** [comment]
>those all have intentional stock limitations on all of their items, leaving some people unable to actually get the item they want because cargo never restocks the things

You obviously haven't seen how players interact with the machines =p

**Comment by tigercat2000** [comment]
> You obviously haven't seen how players interact with the machines =p

whatever, the point is, this is useful for more than a few vendors, especially with it's whitelisting/job whitelisting capability

**Comment by fludd12** [comment]
:potable_water::heart::cat:
Thank you, Tiger!

**Comment by KasparoVy** [comment]
Eyepatch to complete the y'arr matey?

**Comment by Dumbdumn5** [comment]
I really enjoy these changes. Saves time, vendors are there if you want to change clothes to something else, "+1" here. Are the down votes because we already have vendors, or are they because snowflakes or something? If it's for the vendors, yes, you can still get these items in game, just like you can get pretty much anything in the game that's on the station. Given how these things are purely aesthetic with absolutely no bonuses, I have no idea how this could be so negatively impacting that it shouldn't be added. It's a fairly nice way to further customize your character without changing game balance or dicking around with a vendor restock bug, cargo, or just waiting for the ten people in line for the vendors to move while ignoring them are because you were just there to get a new pair of pants so you could start going to work.

**Comment by taukausanake** [comment]
Only thing I would have added to it is a pet collar. That's literally the only thing I use as an accessory. Because reasons

**Comment by Kevak** [comment]
I support the pet collar addition. Don't actually use it but I can't see a reason not to.

**Comment by NTSAM** [comment]
OOH OOH
COWBOY HATS

**Comment by Iamgoofball** [comment]
its 3000 fucking lines and sql changes for what could be accomplished with some quick vendor code adjustments

**Comment by Spacemanspark** [comment]
Who cares.

**Comment by NTSAM** [comment]
tbh it's inconvenient as shit to spend 5 minutes each round having to walk from one edge of the station to another, waiting for the crew to get into their own clothes, then having to use an exploit to refill the machine, then finally scrolling down to find your item, then changing into the item and walking back when you could just spawn with them

**Comment by KasparoVy** [comment]
I don't really care either way. I figure there's no real negative impact to having this implemented and that its existence will grant convenience. But that's not to say that we shouldn't make "some quick vendor code adjustments" for the sake of improvement, either, but that's for other PRs

**Comment by fludd12** [comment]
It occurs to me that this could also be used as a refactor for custom fluff items, where instead of the item being coded for a specific character, they could be added to the user's general loadout.

**Comment by tigercat2000** [comment]
@Iamgoofball 
> its 3000 fucking lines and sql changes for what could be accomplished with some quick vendor code adjustments

it's exactly 457 lines actually added/changed
174 of those lines are literally just item defines for the menu

the other 2500 are from line ending fuckups and moving preferences.dm file around

SQL changes are hardly a big deal, they have to be done for literally any preference changes since we don't use savefiles.

**Comment by Iamgoofball** [comment]
why the hell do you not use savefiles, they work perfectly fine

@NTSAM aw boo hoo you have to actually leave your department to play dress up, what a tragedy

also thanks to admitting abusing a HREF exploit to restock machines, I'll make sure your ban is in place soon

**Comment by NTSAM** [comment]
11 out of 10 people use it and 0 people have ever been penalized for it.

**Comment by Iamgoofball** [comment]
why don't you post details about how to do the HREF exploit because it needs patched ASAP, I can generate infinite of very gamebreaking things using it

**Comment by NTSAM** [comment]
because i cba rn

**Comment by Iamgoofball** [comment]
yeah no you're withholding exploit abuse information so you can keep abusing it locally

@Regen1 if you want to ban this guy for literally abusing game exploits to generate items that'd be swell, as this same exploit would apply to all vendors including ones that hold overpowered items but in small quantities

**Comment by Spacemanspark** [comment]
Goofball we all know what bug he's talking about lmao.

**Comment by NTSAM** [comment]
👍

**Comment by tigercat2000** [comment]
@Iamgoofball 
> why the hell do you not use savefiles, they work perfectly fine

because it makes it much harder to actually moderate and see data about characters
they're all just lumped into one proprietary byond file

> also thanks to admitting abusing a HREF exploit to restock machines, I'll make sure your ban is in place soon

Don't pretend you have literally any power whatsoever with the staff. You have absolutely 0 basis.

**Comment by Iamgoofball** [comment]
@Spacemanspark I don't see an issue report up containing details on HREF exploits for vending machines.

@Fox-McCloud you should do some R&D on this exploit since NT-Sam refuses to say how it works and the circlejerk is real

@tigercat2000 what I can do is spam every admin with this guy's self admitted exploit abuse

**Comment by NTSAM** [comment]
okay so first of all it only works on machines that use the restock modules and second of all i don't know of any non-admin vendor machines that use the restocking modules and finally why the fuck do you even care, i have never seen you on the server ever.

it happens when you deconstruct a vendor machine and rebuild it, it just poofs in more stuff.

**Comment by tigercat2000** [comment]
@Iamgoofball if you intend to actually *spam* admins he isn't the one facing anything

**Comment by VampyrBytes** [comment]
@Iamgoofball 
I can pretty much guarantee that if you start spamming admins someone will get a ban, but it won't be @NTSAM - I can think of at least one admin that would really appreciate you giving him a reason to ban you

**Comment by Iamgoofball** [comment]
@NTSAM Thanks, I'll have a bugfix up by tonight.

**Comment by NTSAM** [comment]
k

**Comment by fludd12** [comment]
I was under the assumption that the vending machine restock was intentional.
Kind of like how you have to cycle out the stock containers.

**Comment by Gummin** [comment]
I'm really certain it is yeah.

**Comment by ProperPants** [comment]
I was under the assumption the stocking units were bugged. IMO, it should show all stock at once, like
>Mad Scientist Labcoat (3) (VEND)

but instead it does whatever garbage it currently does.

**Comment by Vivalas** [comment]
Why the hell is there so much salt as to adding such a useful person by someone who barely even plays this codebase? if anything adding the ability to spawn with items at round start is the more proper and less snowflakey way of doing this than creating a bajillion fucking vending machines.

**Comment by fludd12** [comment]
Because, as much as I disagree with almost every single balance change and removal of "bulk" that Goofball makes, it is undeniable that his code is well-made. Many maintainers grudgingly agree, so his opinion will generally be considered, no matter how often he attempts to fan fires.

**Comment by Iamgoofball** [comment]
the way I see it and why it #triggers me so much:

preferences code is B A D
the UI for it is B A D
adding onto it is B A D

do not read preferences.dm posts
ignore all preferences.dm posts
block all preferences.dm posts

**Comment by Dumbdumn5** [comment]
@Iamgoofball This exploit has been well known and publicly discussed numerous times. That bug that can apparently generate game breaking objects only works on vendors with restocking units, as said above, those being food vendors, clothing vendors, the bartender's booze-o-mat, and cigarette vendors. How you could make it reproduce something more than alcohol and a lighter that could do any massive damage? I'd like to know specifically so I could look for it while observing for grief or other cases where that exploit might greatly change a round.

On your comment above in terms of spamming administrators, that's a way to get a permanent ban quickly, due to harassment of an administrator or the administrative body in general.

I see continual negativity from your posts in a number of PRs on the para github, and you've already been warned for it. Please quit acting detrimentally towards other users due to opinion and simply state what you dislike, rather than talking like this 

> aw boo hoo you have to actually leave your department to play dress up, what a tragedy

Provide constructive criticism as to why you dislike something and leave it at that without getting hostile or vitriolic towards others who don't share your views. If somebody makes a claim that you disagree with, state evidence to the contrary without insulting the other side. Stating that something is bad without stating why does not explain why it is actually bad, just that it is bad in some way. Your contributions are appreciated, but your attitude is not.

**Comment by Fruerlund** [comment]
LaceUp shoes.

**Comment by AceMcLazer** [comment]
I've wanted this in the game for so long~

**Comment by Keilden** [comment]
Add hats, many hats. TF 2 level of hats.

**Comment by FlattestGuitar** [comment]
Personally, I don't like the system. Having items dependent on the role you play will be messy, making people have to switch things around if they want to play more than one department.

I know it's a lot to ask for, but making all jobs have their own groups of items you can equip would make it all a lot less messy to deal with and make it all more customizable.

**Comment by tigercat2000** [comment]
@FlattestGuitar I was considering making another column that states job restrictions

**Comment by FlattestGuitar** [comment]
That would still be a great improvement. If you want to bother, you have all my :+1:.

**Comment by Bawhoppen** [comment]
@Iamgoofball stop fucking with the paradise people our loadouts are going to be completely dfferent

**Comment by Iamgoofball** [comment]
we aren't adding loadouts???

**Comment by Bawhoppen** [comment]
we are (after the freeze)

**Comment by taukausanake** [comment]
Would it also be possible to include suspenders like what the mime and clown wear, or are those going to stay specific to them only?

File: code/modules/client/preference/loadout/gear_tweaks.dm [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+/datum/gear_tweak/color/New(var/list/colors)
+	valid_colors = colors
+	..()
+
+/datum/gear_tweak/color/get_contents(var/metadata)
+	return "Color: <font color='[metadata]'>&#9899;</font>"
+
+/datum/gear_tweak/color/get_default()
+	return valid_colors ? valid_colors[1] : COLOR_GRAY
```
Comment (Signmark) [inline code/modules/client/preference/loadout/gear_tweaks.dm]: COLOR_GRAY not defined

File: code/modules/client/preference/loadout/loadout_uniform.dm [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+	slot = slot_w_uniform
+	sort_category = "Uniforms and Casual Dress"
+
+/datum/gear/uniform/skirt
+	display_name = "skirt"
+
+/datum/gear/uniform/skirt/New()
+	..()
+	var/list/jumpskirts = list()
+	for(var/jump in subtypesof(/obj/item/clothing/under1))
```
Comment (Signmark) [inline code/modules/client/preference/loadout/loadout_uniform.dm]: i think it's /obj/item/clothing/under, not /obj/item/clothing/under1

File: code/modules/client/preference/loadout/loadout_uniform.dm [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+		if(findtext(path2text(jump), "skirt"))
+	gear_tweaks += new/datum/gear_tweak/path(sortAssoc(skirt_list))
```
Comment (Signmark) [inline code/modules/client/preference/loadout/loadout_uniform.dm]: i think all skirts must be under the /obj/item/clothing/under/dress
and then
```
	var/list/dress_list = list()
	for(var/dresses in (typesof(/obj/item/clothing/under/dress)))
		var/obj/item/clothing/under/dress/dress_type = dress
		dress_list[initial(dress_type.name)] = dress_type
	gear_tweaks += new/datum/gear_tweak/path(sortAssoc(dress_list))
```

Comment (Signmark) [inline code/modules/client/preference/loadout/loadout_uniform.dm]: oh, you doing this another way, sorrrrrrrrrrrry

**Comment by Signmark** [comment]
You can use karma instead of "loadout points".
Buy loadout things in karmashop => now you can select it in the loadout.
I think it will make karmashop more valuable thing

File: code/modules/client/preference/loadout/gear_tweaks.dm [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+		return metadata
+
+/datum/gear_tweak/reagents/tweak_item(var/obj/item/I, var/list/metadata)
+	if(metadata == "None")
+		return
+	if(metadata == "Random")
+		. = valid_reagents[pick(valid_reagents)]
+	else
+		. = valid_reagents[metadata]
+	I.reagents.add_reagent(., I.reagents.get_free_space())
```
Comment (Signmark) [inline code/modules/client/preference/loadout/gear_tweaks.dm]: `I.reagents.add_reagent(., I.reagents.maximum_volume - I.reagents.total_volume)`

File: code/modules/client/preference/loadout/loadout_uniform.dm (hunk 7 above)
Comment (Signmark) [inline code/modules/client/preference/loadout/loadout_uniform.dm]: oh i see it now, you made var/list/jumpskirts, but used skirt_list instead, which is not defined

**Comment by Spacemanspark** [comment]
No.

**Comment by Signmark** [comment]
@Spacemanspark No...?

**Comment by Spacemanspark** [comment]
Karma doesn't need to be used for something so trivial as spawning in with a jumpsuit.

**Comment by tigercat2000** [comment]
@Signmark nah, it's supposed to be free shit

**Comment by taukausanake** [comment]
I like the idea of using the karma shop more. Sure, maybe clothing options is free shit but I'd like to work for things. Clothing would be nothing more than one or two karma points and even if karma doesn't have any actual value outside of itself it would be nice to see it get more use

**Comment by tigercat2000** [comment]
it's not using the karma shop

**Comment by AceMcLazer** [comment]
add Polaris's short shorts :3

**Comment by Twinmold93** [comment]
We could definitely use more options, and Polaris's clothing options is wonderful. Also helps make some items that are less stocked, like cards, be available so people can't just run off with the few on station. Because I like having those extra items.

**Comment by Spacemanspark** [comment]
I still want that blue hoodie.

**Comment by tigercat2000** [comment]
@AceMcLazer 
> And, communicators. @tigercat2000 Because. They're wicked.

fuuuuuuuuuck no, ghosts do not get to talk to people

**Comment by Spacemanspark** [comment]
Then remove the ghost functionality from it. If you glance at Polaris' stuff, they've actually turned it into something that's pretty neat.

**Comment by fludd12** [comment]
I'd prefer we get this PR off the ground before making more item requests! The system was made for future additions, after all.

**Comment by Spacemanspark** [comment]
Very true, ya.

**Comment by tigercat2000** [comment]
i'll probably fix all the merge conflicts and finish skirts tonight

**Comment by tigercat2000** [comment]
@Fox-McCloud @TheDZD @Aurorablade 
Going to ask for final review of this, especially the locker stuff, not sure if I missed any or put them in the wrong place.

Other than that, @Necaladun need your word on if initial loadout points should be reduced.

**Comment by Aurorablade** [comment]
well everything looks good to me..still want a bandana though..but thats a small small issue..very very tiny...

**Comment by tigercat2000** [comment]
@Aurorablade can always add more later

File: code/modules/client/preference/loadout/gear_tweaks.dm (hunk 5 above)
Comment (tigercat2000) [inline code/modules/client/preference/loadout/gear_tweaks.dm]: it's defined now :tiger:

**Comment by AceMcLazer** [comment]
@tigercat2000 does that big bold text of "feel free to suggest other items from polaris" stand? From what I can tell you're finishing this PR up soon?

**Comment by tigercat2000** [comment]
@AceMcLazer I decided that it'd be best to leave it as-is with items and add more after the system in general is added. Any suggestions here might get added in a later PR, though, depends when I have some free time.

**Comment by tigercat2000** [comment]
@TheDZD @Fox-McCloud dnm tag can be removed :tiger:

**Comment by TheDZD** [comment]
Looks good to me, just need to wait for a patch to merge this.

**Comment by TheDZD** [comment]
Merge conflicts, also needs changelog.

**Comment by tigercat2000** [comment]
@TheDZD conflicts fixed

**Comment by Signmark** [comment]
Still my opinion - Loadout is cool feature but it looks bad on Polaris. There is no need in this DESCRIPTIONS FOR ITEMS, it need preview for items like it done in this screen 
image
So since there is no need for descriptions you can make buttons for each loadout slot like "Set Head slot", "Set Gloves Slot". Click on this buttons - and you going to choose one item from the list, like it done with species in character setup
image
I hope you'll understand what i suggesting

**Comment by fludd12** [comment]
Oh no more conflicts
Somehow
Also, @Signmark I'm pretty sure that would require a recode of our preview system, as well as a recode of this system, which would suck for poor :tiger:

**Comment by tigercat2000** [comment]
conflicts fixed

**Comment by tigercat2000** [comment]
i am going to stab @Fox-McCloud in his tail

**Comment by tigercat2000** [comment]
HYPE :train: HYPE :train:

**Comment by LittleBigKid2000** [comment]
H Y P E
Y P E
P E
E

**Comment by LittleBigKid2000** [comment]
@tigercat2000 But why are skirts 3 points?

**Comment by tigercat2000** [comment]
@LittleBigKid2000 because no matter how many job skirts you select, it _only_ costs 3 points

**Comment by LittleBigKid2000** [comment]
@tigercat2000 Seems I can only select one at a time, though.

**Comment by tigercat2000** [comment]
@LittleBigKid2000 strange, you should be able to select every job skirt at once for only 3 points

**Comment by Bawhoppen** [comment]
the PR that keeps on giv- spamming my email

**Comment by PJB3005** [comment]
You can disable email notifications you know.

**Comment by Bawhoppen** [comment]
i'm too deep in it now to do that

**Comment by Lianovskiy** [comment]
Нужно добавить сушилку наконец, а то это невозможно. Трубку табаком забивать - херня полная.

**Pull request merged**